### PR TITLE
MNW-276 Refactor config

### PR DESCRIPTION
**Pull request by loosefer**
- removed useless infrastructure files (dockerfiles, sh scripts, etc)
- removed useless near-wallet/utilities packages
- removed feature-flags package
- refactor config to TypeScript, make single entry point for all config variables

### CODE REVIEW

File: packages/frontend/src/hooks/performBuyOrSellUSN.js [hunk 1]
```diff
@@ -55,13 +55,13 @@ export const usePerformBuyOrSellUSN = () => {
         const account = await wallet.getAccount(accountId);
         const usnContract = new nearApiJs.Contract(
             account,
-            USN_CONTRACT,
+            CONFIG,
```
Comment (PinkiNice) [inline packages/frontend/src/hooks/performBuyOrSellUSN.js:58]: Shouldn't there be a string? But object is passed

Comment (NotEternal) [reply packages/frontend/src/hooks/performBuyOrSellUSN.js:58]: Seems I didn't delete this file. As I can see we're not using it anywhere, so let's delete it.

**Comment by NotEternal** [comment]
Empty?

`packages/frontend/src/components/wallet/ExploreApps.js`

File: packages/frontend/src/redux/slices/multiplier/index.js [hunk 2]
```diff
@@ -2,7 +2,7 @@ import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
 import merge from 'lodash.merge';
```
Comment (NotEternal) [inline packages/frontend/src/redux/slices/multiplier/index.js:2]: `slices/multiplier/index.js`

The same as for `performBuyOrSellUSN.js`. I consider we can delete it from the state. Need to double check usage of these files.
Just a notice, we can do it later.

File: packages/frontend/tsconfig.json [hunk 3]
```diff
@@ -33,4 +34,4 @@
     "../utilities",
```
Comment (NotEternal) [inline packages/frontend/tsconfig.json:34]: We don't need it here anymore:

```json
"../feature-flags",
"../utilities",
```

File: packages/frontend/.babelrc [hunk 4]
```diff
@@ -4,5 +4,11 @@
         "@babel/preset-react"
     ],
     "plugins": [
+        [
+            "@babel/plugin-transform-react-jsx",
```
[the remark this answers is not in the document]
Comment (loosefer) [reply packages/frontend/.babelrc:8]: Change removed

File: packages/frontend/src/redux/slices/multiplier/index.js (hunk 2 above)
Comment (loosefer) [reply packages/frontend/src/redux/slices/multiplier/index.js:2]: @NotEternal 
Kindly create a task with this suggestion.

**Pull request merged**